app/views/_layouts/content-page.nunjucks

```diff
@@ -9,30 +9,34 @@
       <div class="grid-row">
 
         <div class="column__two-thirds">
-          <section>
-            {% block main_content %}{% endblock %}
-          </section>
-
-          {% if gpCallout %}
-            <section class="callout callout__info">
-              <h2>See your GP if</h2>
-
-              {% block gp_callout %}{% endblock %}
+          <div class="layout-measure">
+            <section>
+              {% block main_content %}{% endblock %}
             </section>
-          {% endif %}
-        </div>
 
-        <div class="column__one-third column__right">
-          {% block aside_content %}
-            {% if emergencyCallout %}
-              <section class="callout callout__alert callout__overlap">
-                <h2>Call 999 or go to <abbr title="Accident and Emergency">
-                  A&amp;E</abbr> if you</h2>
+            {% if gpCallout %}
+              <section class="callout callout__info">
+                <h2>See your GP if</h2>
 
-                {% block aside_callout %}{% endblock %}
+                {% block gp_callout %}{% endblock %}
               </section>
             {% endif %}
-          {% endblock %}
+          </div>
+        </div>
+
+        <div class="column__one-third-right">
+          <div class="layout-measure">
```

Comment:
Can this class be applied to the callout in this instance rather than requiring another wrapping element?